packages/core/strapi/lib/Strapi.js

```diff
@@ -117,7 +117,7 @@ class Strapi {
     this.cron = createCronService();
     this.telemetry = createTelemetry(this);
     this.requestContext = requestContext;
-
+    this.ee = ee;
```

Comment:
I thought we wanted to remove the ee access from the strapi global but maybe I missed something, @fdel-car didn't you removed this in your last PR? :thinking:

Reply:
It was never there in the first place, if you really wanna add it it must not be configurable like it is now in the PR. Otherwise it becomes trivial to replace it.

Reply:
the ee object is frozen https://github.com/strapi/strapi/blob/chore/ee-seats/packages/core/strapi/ee/index.js#LL160C18-L177C4

Would this be enough to prevent mutations on this ee?